src/content/content.controller.ts

```
@@ -0,0 +1,173 @@
+  @ApiParam({ name: 'id', description: 'Content UUID' })
+  @ApiBody({ type: UpdateContentDto })
+  @ApiResponse({
+    status: 200,
+    description: 'Content updated successfully',
+  })
+  @ApiBadRequestResponse({ description: 'Bad Request' })
+  @ApiNotFoundResponse({ description: 'Content not found' })
+  @ApiInternalServerErrorResponse({ description: 'Internal Server Error' })
+  @UsePipes(new ValidationPipe({ transform: true, whitelist: true }))
+  async update(
+    @Param('id') id: string,
```

ESLint

[error] Replace `'id'` with `"id"`

(prettier/prettier)